src/hooks/events.ts

```diff
@@ -25,13 +29,24 @@ export interface VenueEventsData {
 export const useVenueEvents: ReactHook<VenueEventsProps, VenueEventsData> = ({
   venueIds,
 }) => {
+  const firebase = useFirebase();
+
   const {
     loading: isEventsLoading,
     error: eventsError,
     value: events = emptyArray,
   } = useAsync(async () => {
-    if (!venueIds) return emptyArray;
+    //load from gs
+    const storage = firebase.storage();
+    const url = await storage
+      .ref()
+      .child(SCHEDULE_LOAD_FROM_GS)
+      .getDownloadURL();
+    return fetch(url).then((res) => res.json());
```

Comment:
Also, is there a way to filter by `venueIds` array is it is available? An issue has come up in the posterhall where the live events aren't filtered to only those specified by parentId (== posterhall). Something like?: 

`events.filter((event) => venueIds.includes(event.venueId))`

Reply:
Yes. I should filter them.. I just did that. I've also updated to store the cache in the redux store instead of relying on fetch() to return the cached output.

Reply:
Thanks! Just tested, and seems to work smoothly in the posterhall now